src/main/java/com/cloudbees/jenkins/plugins/advisor/AdvisorGlobalConfiguration.java

```diff
@@ -343,21 +379,19 @@ public String getDisplayName() {
       return Messages.Insights_DisplayName();
     }
 
+    // Used from AdvisorGlobalConfiguration/index.jelly
     public FormValidation doCheckAcceptToS(@QueryParameter boolean value) {
       if (!value) {
         return FormValidation.error("Accepting our Terms and Conditions is mandatory to use this service.");
       }
       return FormValidation.ok();
     }
-
+    
+    // Used from AdvisorGlobalConfiguration/index.jelly
     public FormValidation doCheckEmail(@QueryParameter String value) {
       return EmailValidator.validateEmail(value);
     }
-
-    public FormValidation doCheckCc(@QueryParameter String value) {
-      return EmailValidator.validateCC(value);
-    }
-
```

Comment:
How do you check from UI the CC email addresses?

Reply:
I don't it's something I forgot to remove

Reply:
@fcojfernandez They are individually verified in the Recipient